ibis/omniscidb/client.py

```diff
@@ -905,6 +905,44 @@ def drop_table(self, table_name, database=None, force=False):
         self._execute(statement, False)
         self.set_database(_database)
 
+    def add_column(self, table_name, column_name, data_type):
```

Comment:
I think it is possible to add several columns at a time through giving a list of column names and a list of data types. How do you like it? Can you implement it?

Reply:
It can be implemented, but I think it's enough.

Reply:
It will allow getting more performance because we will send and execute one query while adding several columns. It's not so difficult to do so I suggest doing it. It's up to you.

Reply:
If we see more performance from it the I will implement method for some columns.